spec/integration/elasticsearch/integration_spec.rb

```diff
@@ -23,7 +23,7 @@ class MyModel < Hashie::Mash
     object = MyModel.new
     stub_elasticsearch_client
 
-    expect { object.__elasticsearch__.index_document }.to raise_error(NoMethodError)
+    expect { object.__elasticsearch__.index_document }.to raise_error(NameError)
```

Comment:
Hmm, this changing surprises me. Any idea why it changed?

Reply:
It changed with the most recent major version release of the elasticsearch library

Reply:
I am cool with this change, but maybe we can explicitly depend on `elasticsearch > X.Y` in Gemfile for clarity?